instabug/README.md

```diff
@@ -0,0 +1,36 @@
+# Instabug integration
+## Overview
+
+[Instabug][1] is a mobile-focused platform that empowers mobile teams to monitor, prioritize, and debug performance and stability issues throughout the mobile app development lifecycle.
+
+The Instabug dashboard widget in Datadog helps you monitor your overall app health and how users perceive your app's performance with just one number, the App Apdex. The widget provides:
+- The overall App Apdex score
+- App Apdex overtime
+- A breakdown of the sessions into four buckets (`Satisfying`, `Tolerable`, `Frustrating` or `Crashing` sessions)
+- The five most recent bug reports and the total number of new reports
+
+
+## Setup
+1. If you haven't already, [Sign up for Instabug][2] for free and follow the steps to integrate the SDK into your app. Ensure that you have also joined the [Datadog Apps beta][5].
```

Comment:
```suggestion
1. If you haven't already, [Sign up for Instabug][2] for free and follow the steps to integrate the SDK into your app.
```
Customers do not need to join the Datadog Apps beta program. They will get access to the app/UI extension when they install the integration.

Reply:
Could you also clarify the steps that customers are required to follow? I don't see a link to instructions anywhere for the SDK.